src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -3158,3 +3166,323 @@ components:
+              type: string
+    ProgrammaticScryptoSborValueU16:
+      allOf:
+        - $ref: "#/components/schemas/ProgrammaticScryptoSborValue"
+        - type: object
+          required:
+            - value
+          properties:
+            value:
+              type: string
+    ProgrammaticScryptoSborValueU32:
+      allOf:
+        - $ref: "#/components/schemas/ProgrammaticScryptoSborValue"
+        - type: object
+          required:
+            - value
+          properties:
+            value:
+              type: string
+    ProgrammaticScryptoSborValueU64:
+      allOf:
+        - $ref: "#/components/schemas/ProgrammaticScryptoSborValue"
+        - type: object
+          required:
+            - value
+          properties:
+            value:
+              type: string
+    ProgrammaticScryptoSborValueU128:
+      allOf:
+        - $ref: "#/components/schemas/ProgrammaticScryptoSborValue"
+        - type: object
+          required:
+            - value
+          properties:
+            value:
+              type: string
+    ProgrammaticScryptoSborValueString:
+      allOf:
+        - $ref: "#/components/schemas/ProgrammaticScryptoSborValue"
+        - type: object
+          required:
+            - value
+          properties:
+            value:
+              type: string
+    ProgrammaticScryptoSborValueEnum:
+      allOf:
+        - $ref: "#/components/schemas/ProgrammaticScryptoSborValue"
+        - type: object
+          required:
+            - discriminator
+            - fields
+          properties:
+            discriminator:
+              type: string
+            fields:
+              type: array
+              items:
+                $ref: "#/components/schemas/ProgrammaticScryptoSborValue"
+    ProgrammaticScryptoSborValueArray:
+      allOf:
+        - $ref: "#/components/schemas/ProgrammaticScryptoSborValue"
+        - type: object
+          required:
+            - element_kind
+            - elements
+          properties:
+            element_kind:
+              $ref: "#/components/schemas/ProgrammaticScryptoSborValueTypeKind"
+            element_type_name:
+              type: string
+            elements:
+              type: array
+              items:
+                $ref: "#/components/schemas/ProgrammaticScryptoSborValue"
+    ProgrammaticScryptoSborValueTuple:
+      allOf:
+        - $ref: "#/components/schemas/ProgrammaticScryptoSborValue"
+        - type: object
+          required:
+            - fields
+          properties:
+            fields:
+              type: array
+              items:
+                $ref: "#/components/schemas/ProgrammaticScryptoSborValue"
+    ProgrammaticScryptoSborValueMap:
+      allOf:
+        - $ref: "#/components/schemas/ProgrammaticScryptoSborValue"
+        - type: object
+          required:
+            - key_value_kind
+            - value_value_kind
+            - entries
+          properties:
+            key_value_kind:
```

Comment:
You're missing non-required/nullable `key_type_name` and `value_type_name` here

Reply:
Added.